src/query/plan/pretty_print.cpp

```diff
@@ -188,12 +187,28 @@ bool PlanPrinter::PreVisit(query::plan::Filter &op) {
   return false;
 }
 
+bool PlanPrinter::PreVisit(query::plan::EdgeUniquenessFilter &op) {
+  WithPrintLn([&](auto &out) {
+    out << "* EdgeUniquenessFilter [";
+    utils::PrintIterable(out, op.previous_symbols_, ", ", [](auto &out, const auto &sym) { out << sym.name(); });
+    out << " != " << op.expand_symbol_.name() << "]";
+  });
+  return true;
+}
```

Comment:
I suggest implementing `PlanPrinter::Previsit` for this operator with a `ToString` method, consistently with other operators that have detailed info (see [PR](https://github.com/memgraph/memgraph/pull/1204)).